docs/Contributing.md

```diff
@@ -3,6 +3,12 @@
 
 Wallet Core implements the cryptographic functionality of blockchains. This includes elliptic curve cryptography, hashing, address derivation and transaction signing. However it *does not* implement other aspects like networking and UI. Wallet core behaves like a black box for higher level users like Trust Wallet; it takes inputs from the blockchain and the user (for instance UTXOs, private keys, etc.) and produces an output (like a signed and encoded transaction).
 
+### How does Trust Wallet interact with Wallet Core
+
+Take Ethereum as an example:
+
+<img src="wallet-core.png">
```

Comment:
Can you render it at double the size? It looks grainy on a Retina display.

Reply:
how about now?

Reply:
No, still blurry. Not sure if it's fixable...